control-plane/rest/service/src/v0/mod.rs

```diff
@@ -46,6 +48,13 @@ pub(crate) fn core_grpc<'a>() -> &'a CoreClient {
         .expect("gRPC Core Client should have been initialised")
 }
 
+/// Get Json gRPC Client
+pub(crate) fn json_grpc<'a>() -> &'a JsonGrpcClient {
```

Comment:
should return an option in case the server is not configured with the json grpc agent?

Reply:
Yes. If a call is being made without the configuration, returning the below.
```
{
  "details": "",
  "message": "JsonGrpc Service is not configured/running.",
  "kind": "Unavailable"
}
```